src/components/MatchPage/MatchCard.js

```diff
@@ -1,31 +1,53 @@
 import React, { useState,useEffect } from 'react';
 import { db } from '../../services/firebase';
 import './MatchCard.css';
+import SingleMatchPopup from './SingleMatchPopup/SingleMatchPopup';
+import { Button,Typography } from '@material-ui/core';
+
 function MatchCard(props) {
-    const [info, setInfo] = useState({});
+    const [info, setInfo] = useState({name: "Loading..."});
+    const [showPopup, setShowPopup] = useState(false);
+    const openPopup = () => setShowPopup(true);
+    const closePopup = () => setShowPopup(false);
+    let user_info = {};
 
     useEffect(() =>{
-        const userRef = db.ref('Users/'+props.user_id+'/personalInfo'); 
-        let user_info = {}
-    
+        const userRef = db.ref('Users/'+props.user_id);
```

Comment:
same thing here, i don't think we need to call to the database here, since we have it in userData from authContext. The userData value is updating when the database updates as well (i think), since it's doing the db.ref(...).on('value') thing.